include/votca/xtp/segment.h

```diff
@@ -0,0 +1,199 @@
+/*
+ *            Copyright 2009-2018 The VOTCA Development Team
+ *                       (http://www.votca.org)
+ *
+ *      Licensed under the Apache License, Version 2.0 (the "License")
+ *
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+
+#ifndef __VOTCA_XTP_SEGMENT_H
+#define	__VOTCA_XTP_SEGMENT_H
+
+#include <map>
+#include <vector>
+
+#include <votca/tools/vec.h>
+
+namespace votca { namespace xtp {
+
+class Atom;
+class PolarSite;
+class APolarSite;
+class Fragment;
+class SegmentType;
+class Topology;
```

Comment:
Can we use the topology from csg?

Reply:
I think it already does I just have not imported the topology class from ctp yet.